toolbox: add pages (#8280)
tldr-pages/tldr: pages/linux/toolbox-init-container.md

# toolbox init-container

> > Initialize a running container.

Reviewer: This seems like important information
```suggestion
> Initialize a running `toolbox` container.
> This command is not expected to be directly invoked by humans, and cannot be used on the host.
```

> More information: <https://www.mankier.com/1/toolbox-init-container>.

> - Pass gid as the user's numerical group ID from the host to the `toolbox` container:

Reviewer: This command initializes the container with this specific GID, right? Then it should be more like:
```suggestion
- Initialize a running container and pass the user's GID from the host to the `toolbox` container:
```
But is this command even possible, since it cannot be executed on the host? And which `toolbox` container is it passed to?

Author: I haven't used a init-container before so I don't know the feasibility of the command.

`toolbox init-container --gid {{gid}}`

- Create a user inside the toolbox container whose login directory is home (Required):

`toolbox init-container --home {{home}}`

- Create a user inside the `toolbox` container whose login shell is shell (Required):

`toolbox init-container --shell {{shell}}`

- Create a user inside the `toolbox` container whose numerical user id is uid (Required):

`toolbox init-container --uid {{uid}}`

- Create a user inside the `toolbox` container whose login name is login (Required):

> `toolbox init-container {{user}}`

Reviewer: I'd maybe group all those commands and write something like `Initialize a running toolbox and specify all required arguments:`

# toolbox init-container

> Initialize a running `toolbox` container.

> > This command is not expected to be directly invoked by humans, and cannot be used on the host.

Reviewer 2: While it sounds sci-fi, I am not happy with this wording.

```suggestion
> This command is should not be executed by the user, and cannot be run on the host.
```

Author: This looks better

> More information: <https://manned.org/toolbox-init-container.1>.

- Initialize a running toolbox:

> `toolbox init-container --gid {{gid}} --home {{home}} --home-link --media-link --mnt-link --monitor-host --shell {{shell}} --uid {{uid}} --user {{user}}`

Reviewer: Are the `--...-link` and `-monitor-host` arguments required? If not you could maybe make another example, so that the first one is as minimal as it can be. (But only if it makes sense without them).

Author: I haven't used init containers before. I referred to this from the synopsis at <https://github.com/containers/toolbox/blob/main/doc/toolbox-init-container.1.md> ,

Author: @marchersimon maybe we can skip this page. I will remove it if necessary. It seems a bit confusing.

Reviewer: I think we can keep the page. At least we mention that it's not supposed to be used by humans.

Regarding the rename to `toolbx`, usually we add an alias page for this. However, in the blog they state that this alias doesn't exist yet:

> So, those will gradually happen over time with all the necessary compatibility aliases and such.

You could add an alias page once the `toolbx` binary exists if you want to.

Author: Sure, then I will leave this page as it is. In future when it becomes toolbx I will add an alias page.
